data/sql/grid_transformation_custom.sql

```diff
@@ -84,3 +84,38 @@ INSERT INTO "grid_transformation" VALUES(
     NULL,
     'EPSG','8666','Geoid (height correction) model file','Icegeoid_ISN2016.gtx',
     NULL,NULL,NULL,NULL,NULL,NULL,NULL,0);
+
+-- Norway
+
+ INSERT INTO "grid_transformation" VALUES(
+    'PROJ','EPSG_4937_TO_EPSG_5776','ETRS89 to NN1954 height',
+    NULL,NULL,
+    'EPSG','9665','Geographic3D to GravityRelatedHeight (tif)',
+    'EPSG','4937', -- source CRS (ETRS89 3D)
+    'EPSG','5776', -- target CRS (NN1954 height)
+    'EPSG','1352', -- area of use: Norway - Onshore
+    NULL,
+     'EPSG','8666','Geoid (height correction) model file','href2008a.tif',
+    NULL,NULL,NULL,NULL,NULL,NULL,NULL,0);
+
+ INSERT INTO "grid_transformation" VALUES(
+    'PROJ','EPSG_5776_TO_EPSG_5941','ETRS89 to NN2000 height',
+    NULL,NULL,
+    'EPSG','9665','Geographic3D to GravityRelatedHeight (tif)',
+    'EPSG','4937', -- source CRS (ETRS89 3D)
+    'EPSG','5941', -- target CRS (NN2000 height)
+    'EPSG','1352', -- area of use: Norway - Onshore
+    NULL,
+    'EPSG','8666','Geoid (height correction) model file','HREF2018B_NN2000_EUREF89.tif',
+    NULL,NULL,NULL,NULL,NULL,NULL,NULL,0);
+     
+  INSERT INTO "grid_transformation" VALUES(
+     'PROJ','EPSG_5941_TO_EPSG_5776','NN1954 to NN2000 height',
```

Comment:
The name is not consistent with the source and target. It should be 'NN2000 height to NN1954 height' if the values in NNTrans2018B.tif are the valeus to add to a NN2000 height to get a NN1954 one

Reply:
According to the method EPSG 8732 (Vertical offset file) the resources file NNTrans2018B.tif has incorrect signs. I will fix the resosurce file in Proj-data.

By the way, data/sql/grid_alternatives.sql is a better solution. We will focus on implementing the norwegian height transformation that way.

Reply:
> By the way, data/sql/grid_alternatives.sql is a better solution. We will focus on implementing the norwegian height transformation that way.

For proper implementation, you need both a record in grid_transformation (either coming from EPSG, or a custom entry like that), and a record in the grid_alternative table so that PROJ is aware that the resource declared in grid_transformation is also known by PROJ-data, and provide potential renaming from the original resource name to the one used by PROJ-data, URL where to download it, etc...